mmvstr.h

```diff
@@ -127,7 +126,13 @@ with 'let(&' and thus has the same effect as 'let(&'.
 #include <stdio.h>
 
 typedef char* vstring;
-#define vstringdef(x) vstring x = ""
+#define vstring_def(x) vstring x = ""
+#define free_vstring(x) let(&x, "")
+
+/* Free space allocated for temporaries. This is normally called automatically
+  by let(), but it can also be called directly to avoid buildup of temporary
+  strings. */
+void freeTempAlloc(void);
```

Comment:
This affects other artefacts as well.  Difficult to see whether this is correct.

Reply:
Forget about this one.  Possible collisions are mentioned in one of my earlier remarks